### ISSUE TITLE
Open Images MAP test killed

### ISSUE THREAD

**Issue by d3-worgan** [AlexeyAB/darknet#5230]
Hi, 

I am trying to run the detector map test for pjreddies open images detection model, using a subset of the open images test set.

The test runs up to the "rank = 100000+" then is killed. I have tried running on a 6GB GPU and then on a 16GB GPU but both seem to stop at around the same time? 

Does anyone have any ideas how to fix this or if anyone knows of some map test results for YOLOv3 trained on Open Images? I'd like to see the results for each object if possible.

Thanks

**Comment by AlexeyAB** [comment]
* What command do you use?
* Where did you get OpenImages dataset with labels in Yolo format?
* Show screenshot of this error.
* Show content of generated files `bad.list` and `bad_label.list`

**Comment by d3-worgan** [comment]
I used this command:

```
./darknet detector map '/home/darknet/cfg/openimages.data' '/home/darknet/cfg/yolov3-openimages.cfg' '/home/darknet/yolov3-openimages.weights' > /home/darknet/data/oi_map_test/test_results.txt
```
I downloaded the dataset from open images v4 using this [script](https://github.com/d3-worgan/darknet-docker/blob/master/download-multi-together.py), and then fixed the labels indexes for the objects I wanted using this [script](https://github.com/d3-worgan/darknet-docker/blob/master/fix_oi_labels.py).

This is what happens when it is killed.
![Screenshot from 2020-04-13 19-33-38crop](https://user-images.githubusercontent.com/56825720/79150288-66cb1700-7dc0-11ea-8209-8eadcc4dab86.png)

This is where the test was killed
![Screenshot from 2020-04-13 19-35-15crop](https://user-images.githubusercontent.com/56825720/79150114-1eabf480-7dc0-11ea-9b42-f0f71e85b4cc.png)

Where would I find the bad.list and bad_label.list files? I cannot see them in darknet directory.

I have seen this happen before and I have fixed it by either changing batch and subdivisions to 1, or by increasing memory. This last attempt used a tesla v100 with 12 CPUs and 32GB ram but was still killed?
